### ISSUE TITLE
CIS Alarms not working when using name prefix

### ISSUE THREAD

**Issue by ryanoolala** [terraform-aws-modules/terraform-aws-cloudwatch#31]
## Description
CIS Alarms not triggered due to wrong metric name when using `name_prefix` or `use_random_name_prefix` pet names

## Versions

 - Terraform: v1.0.7

 - Provider(s): registry.terraform.io/hashicorp/aws v3.66.0

+ provider registry.terraform.io/hashicorp/external v2.2.0
+ provider registry.terraform.io/hashicorp/local v2.1.0
+ provider registry.terraform.io/hashicorp/null v3.1.0
+ provider registry.terraform.io/hashicorp/random v3.1.0

## Reproduction

Setting name_prefix variable

### Code Snippet to Reproduce

```hcl
module "cis_alarms" {
   source  = "terraform-aws-modules/cloudwatch/aws//modules/cis-alarms"
   version = "~> 2.3"
   alarm_actions     = ["your_sns"]
   name_prefix       = "test-"
}
```
## Expected behavior

Expect that Cloudwatch Alarm to change state when metric changes

## Actual behavior

Metric in cloudwatch alarms are empty an no alarms are triggered, as the prefixed name is different from the cloudwatch metric name

**Comment by ryanoolala** [comment]
related to #14

**Comment by antonbabenko** [comment]
This issue has been resolved in version 2.3.1 :tada:
